### PR TITLE
[zh-cn] Sync reference/setup-tools/kubeadm

### PR DESCRIPTION
**Pull request by tico88612**
- `docs/reference/setup-tools/kubeadm/kubeadm-certs.md`
- `docs/reference/setup-tools/kubeadm/kubeadm-init-phase.md`

### CODE REVIEW

File: content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-certs.md [hunk 1]
```diff
@@ -46,6 +46,7 @@ For more details see [Manual certificate renewal](/docs/tasks/administer-cluster
 {{< tab name="etcd-server" include="generated/kubeadm_certs_renew_etcd-server.md" />}}
 {{< tab name="front-proxy-client" include="generated/kubeadm_certs_renew_front-proxy-client.md" />}}
 {{< tab name="scheduler.conf" include="generated/kubeadm_certs_renew_scheduler.conf.md" />}}
+{{< tab name="super-admin.conf" include="generated/kubeadm_init_phase_kubeconfig_super-admin.conf.md" />}}
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-certs.md:49]: This file is required to enable the ci success.

Comment (tico88612) [reply content/zh-cn/docs/reference/setup-tools/kubeadm/kubeadm-certs.md]: That was a typo.

**Pull request merged**